pkg/cmd/pipelinerun/list_test.go

```diff
@@ -115,6 +116,42 @@ func TestListPipelineRuns(t *testing.T) {
 				"",
 			},
 		},
+		{
+			name:    "limit pipelineruns returned to 1",
+			command: command(t, prs, clock.Now()),
+			args:    []string{"list", "-n", "namespace", "-l", fmt.Sprintf("%d", 1)},
+			expected: []string{
+				"NAME    STARTED          DURATION   STATUS      ",
+				"pr1-1   59 minutes ago   1 minute   Succeeded   ",
+				"",
+			},
+		},
+		{
+			name:    "limit pipelineruns negative case",
+			command: command(t, prs, clock.Now()),
+			args:    []string{"list", "-n", "namespace", "-l", fmt.Sprintf("%d", -1)},
+			expected: []string{
+				"",
+			},
+		},
+		{
+			name:    "limit pipelineruns greater than maximum case",
+			command: command(t, prs, clock.Now()),
+			args:    []string{"list", "-n", "namespace", "-l", fmt.Sprintf("%d", 4)},
+			expected: []string{
+				"",
```

Comment:
Test result set to `""` since error message is sent as Stderr and result is returning `nil`.

```golang
fmt.Fprintf(os.Stderr, "Limit was %d but cannot be greater than %d\n", limit, prslen)
```